app/controllers/v0/profile/telephones_controller.rb

```diff
@@ -9,11 +9,15 @@ class TelephonesController < ApplicationController
       after_action :invalidate_cache
 
       def create
-        write_to_vet360_and_render_transaction!('telephone', telephone_params)
+        write_to_vet360_and_render_transaction!('telephone', delete_params(telephone_params))
```

Comment:
When I read through each one of the controller actions, seeing `delete_params` seems out of place.  No?

Perhaps change the name of `delete_params` to something like `adjust(telephone_params )` or `sanitize(telephone_params )`, `adapt(telephone_params )`, or `set_(telephone_params )`, or something in that vein.  

`delete_params` just seems like you are in fact deleting the params.  When you are really not, you are just customizing one of the values in the params.